src/cascadia/TerminalApp/Jumplist.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Jumplist.h"
+
+#include <Propkey.h>
+
+using namespace winrt::TerminalApp;
+
+//  This property key isn't already defined in Propkey.h, but is used by UWP Jumplist to determine the icon of the jumplist item.
+//  IShellLink's SetIconLocation isn't going to read "ms-appx://" icon paths, so we'll need to use this to set the icon.
+DEFINE_PROPERTYKEY(PKEY_AppUserModel_DestListLogoUri, 0x9F4C2855, 0x9F79, 0x4B39, 0xA8, 0xD0, 0xE1, 0xD4, 0x2D, 0xE1, 0xD5, 0xF3, 29);
+#define INIT_PKEY_AppUserModel_DestListLogoUri                                         \
+{                                                                                      \
+    { 0x9F4C2855, 0x9F79, 0x4B39, 0xA8, 0xD0, 0xE1, 0xD4, 0x2D, 0xE1, 0xD5, 0xF3 }, 29 \
+}
+
+// Method Description:
+// - Updates the items of the Jumplist based on the given settings.
+// Arguments:
+// - settings - The settings object to update the jumplist with.
+// Return Value:
+// - <none>
+void Jumplist::UpdateJumplist(std::shared_ptr<::TerminalApp::CascadiaSettings> settings)

```

Comment:
Considering not passing the whole settings object in, really depends on what we want to put into the jumplist?

Reply:
I think this is fine, tbh.